recipes/brpc/all/conanfile.py

```diff
@@ -0,0 +1,102 @@
+from conans import CMake, ConanFile, tools
+import glob
+import os
+
+
+class BrpcConan(ConanFile):
+    name = "brpc"
+    description = "An industrial-grade RPC framework used throughout Baidu"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/apache/incubator-brpc"
+    topics = ("conan", "brpc", "baidu", "rpc")
+    license = ("Apache-2.0")
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "with_snappy": [True, False],
+        "with_glog": [True, False],
+        "with_thrift": [True, False],
+        "shared": [True, False],
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "protobuf:with_zlib": True,
+        "leveldb:with_snappy": True,
+        "with_snappy": True,
+        "with_glog": True,
+        "with_thrift": False,
+        "shared": False,
+        "fPIC": False,
+    }
+
+    _cmake = None
+
+    def requirements(self):
```

Comment:
You use the `protoc` executable, which should be built for the build system, not the host system.
```suggestion
    _cmake = None

    def build_requirements(self):
        self.build_requires("protobuf/3.9.1")
    
    def requirements(self):
```